RA_Integration.sln

```diff
@@ -19,16 +19,13 @@ EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Analysis|Win32 = Analysis|Win32
-		AnalysisUnicode|Win32 = AnalysisUnicode|Win32
```

Comment:
Noticed this solution config was here, remembered in another PR that we don't need solution-wide unicode configurations.